js/scenes/AppOnboardingScreen/AppOnboardingScreen.js

```diff
@@ -1,16 +1,88 @@
-import React from 'react';
-import { Text } from 'react-native';
-import styles from './styles';
+import React, { Component } from 'react';
+import { ScrollView, View } from 'react-native';
+import PageControl from 'react-native-page-control';
 import AppOnboarding from './../../components/AppOnboarding';
+import { colors, windowHeight, windowWidth } from './../../styles/baseStyles';
+import { styles } from './styles';
 
+class AppOnboardingScreen extends Component {
+  constructor() {
+    super();
 
-const AppOnboardingScreen = () => {
-  return (
-    <AppOnboarding
-      image={require('./../../../assets/images/foodtable.png')}
-      text={'Easily organize potlucks & Share with friends and family'}
-    />
-  );
-};
+    this.state = {
+      currentPage: 0,
+    };
+  }
+
+  onScroll = (event) => {
+    const offsetX = event.nativeEvent.contentOffset.x;
+
+    this.setState({
+      currentPage: Math.floor((offsetX - (windowWidth / 2)) / windowWidth) + 1,
```

Comment:
This is pretty dense. Will it work if you add more steps? If not, maybe make it into a function that is more dynamic.